code/game/objects/effects/effect_system/foam.dm

```diff
@@ -33,8 +33,10 @@
 		STOP_PROCESSING(SSobj, src)
 		sleep(30)
 
-		if(metal)
-			new /obj/structure/foamedmetal(loc)
+		if(metal == 3)
+			new /obj/structure/razorwire(loc)
+		else if(metal)
+			new /obj/structure/foamedmetal(loc)		
```

Comment:
use defines

Reply:
Not smart enough to do that, will work on later.

Reply:
ping me if you need help

Reply:
done